qemu/tests/cfg/hpt.cfg

```diff
@@ -0,0 +1,36 @@
+# Notes:
+#     These test cases are only for HPT guest,which include Power8 guest
+#     and Power8 guest with compatible mode running on Power9 host.
+- hpt:
+    type = hpt_miscellaneous
+    no Host_RHEL.6
+    no RHEL.5 RHEL.6
+    only pseries
+    variants:
+        - increase:
+            mistype = "increase"
```

Comment:
Can we use a more meaningful name for this param?

Reply:
Changed it to sub_type.